.github/workflows/pull-request-build.yaml

```diff
@@ -44,7 +43,12 @@ jobs:
 
       - name: Test
         run: pnpm test
-
+        env:
+          ONE_INCH_API_KEY: ${{ secrets.ONE_INCH_API_KEY }}
+          ONE_INCH_API_VERSION: ${{ secrets.ONE_INCH_API_VERSION }}
+          ONE_INCH_API_URL: ${{ secrets.ONE_INCH_API_URL }}
+          ONE_INCH_ALLOWED_SWAP_PROTOCOLS: ${{ secrets.ONE_INCH_ALLOWED_SWAP_PROTOCOLS }}
```

Comment:
We have different allowed protocols per chain, also those protocols may also have different name depending on chain.

Reply:
Mmm, this is a tricky one then. I mean, we could provide a JSON in the environment variables, but sounds a bit too much. We should discuss how to provide complex config to the different services we are going to have

Reply:
maybe we need env variable per chain `ONE_INCH_ALLOWED_SWAP_PROTOCOLS_ETHEREUM` etc

Reply:
Discussed offline to make a note to investigate this in the future to see how we will provision the config for the different services